addons/ai/stringtable.xml

```diff
@@ -1,109 +1,109 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
single quotes in header is fine for the spec but I tested tabler and it will rewrite to double on save

Reply:
I didn't look into it too hard, but `xml.etree.ElementTree` rewrites them automatically into single quotes.